alpha/external-api-dropwizard/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM openjdk:8-jdk
+
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+ADD . /usr/src/app
```

Comment:
This will copy all the contents of the Dockerfile folder to here, which is not needed. All you want is the jar and yml config.